### ISSUE TITLE
Application signature input does not have assigned item numbers

### ISSUE THREAD

**Issue by petertgiles** [GCTC-NTGC/gc-digital-talent#5498]
Context: Split from #5358

## ❌ Current behaviour (issue)
Application signature input does not have assigned item numbers for alternative navigation. 
### Location: 
Pool application, step 2
### Category: 
Critical Accessiblity Bug
### AT configuration(s):  
Voice Control, Mobile, IOS, Chrome

### How to reproduce
**With Voice Control** 
1.	Begin pool application, advance to step 2
2.	Choose numbered grid overlay. 
3.	Observe there is no number available to focus the signature input 
### Fable Video Clip:
[Signature interaction Issues](https://app.makeitfable.com/clip/63bcd337604482001abf7428) 

## ✔ Expected behaviour (fix)
The user can activate the signature box and input characters
**Potential WCAG Considerations**  [ 2.1.1 Keyboard Navigation](https://www.w3.org/WAI/WCAG22/Understanding/keyboard).

### Acceptance Criteria

- [ ] Signature input can be interacted with using Voice Control on iPhone

**Comment by petertgiles** [comment]
Oops, duplicate of $5357
